### DISCUSSION TITLE
Test Lixee Zlinky la téléinfo plug’n play - Haade.fr

### DISCUSSION THREAD

**Discussion by giscus** [haade-administrator/haade.fr#38] [Comments]
# Test Lixee Zlinky la téléinfo plug’n play - Haade.fr

Test du module Zlinky pour la remontée de la téléinformation spécial linky fabriqué par Lixee et plug’n play, compatible toutes box opensource

https://haade.fr/fr/blog/teleinfo-plug-n-play-zigbee-linky-lixee-zlinky-tic

**Comment by sebastienserre** [comment]
Merci pour l'article.
Une petite précision, Open Source est different de gratuit. Open Source veut simplementn dire que les sources sont dispo pour qui veut le faire soit même ou prendre la base pour l'améliorer et proposer a son tour un produit plus abouti ou différent.
Ceci n'empêche en rien le fait de commercialiser son projet open source... et heureusement, un développeur a aussi besoin de manger.

**Reply by haade-administrator** [reply]
Effectivement j'aurai du noter application opensource par contre je ne parle pas de gratuité

**Comment by algaillard2000-gif** [comment]
Bonjour,
Je viens d'installer ce module sur mon compteur Linky. 
Il s'est connecté immédiatement sur mon réseau Zigbee et clignote bien au rythme de 3 s.
Mon problème est que toutes les informations ne remontent pas, notamment IINST, PTEC, IMAX, PAPP...
Que puis-je faire ?

**Reply by haade-administrator** [reply]
exclure et réinclure dansq un premier temps, quelle service zigbee utilises-tu ( zha, z2m, .. )
